Jenkinsfile

```diff
@@ -91,6 +91,7 @@ try {
           //  But still build all ABI's and run all types of tests. 
           useEmulator = true
           emulatorImage = "system-images;android-29;default;x86"
+          buildFlags = "-PenableLTO=true -PbuildCore=true"
```

Comment:
For traceability/reproducability etc. I would assume that it safest to do release from the prebuilts. Is it safe to to the release from locally build cores? Or don't we get the benefits of optimizations unless we build it?